packages/api/src/external/ehr/athenahealth/command/get-patient.ts

```diff
@@ -46,11 +56,20 @@ export async function getPatientIdOrFail({
     });
     return metriportPatient.id;
   }
-  if (!athenaUrl) throw new Error("AthenaHealth url not defined");
-  const athenaPatient = await getAthenaPatient({
+  if (!athenaEnvironment || !athenaClientKeySecretArn || !athenaClientSecretSecretArn) {
+    throw new Error("AthenaHealth not setup");
+  }
+  const athenaClientKey = await getSecretValueOrFail(athenaClientKeySecretArn, region);
+  const athenaClientSecret = await getSecretValueOrFail(athenaClientSecretSecretArn, region);
+  const api = await AthenaHealthApi.create({
+    threeLeggedAuthToken: accessToken,
+    practiceId: athenaPracticeId,
+    environment: athenaEnvironment as "api" | "api.preview",
```

Comment:
this is the second time I see `"api" | "api.preview"` - let's put this type def in a shared file?

Reply:
Done.